### PR TITLE
Add DbModelSetting

### PR DESCRIPTION
**Pull request by irlirla**
add Db model DbModuleSetting
tune fluentApi
create migration to add ModuleSettings table
create PR and make fixes

### CODE REVIEW

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs [hunk 1]
```diff
@@ -91,6 +91,27 @@ protected override void Up(MigrationBuilder builder)
         {
           table.PrimaryKey($"PK_{DbUnsentEmail.TableName}", x => x.Id);
         });
+
```
Comment (MariyaSin) [inline src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs:94]: you can not add anything to DB when you add via the already applied migration. Create new migration

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs [hunk 2]
```diff
@@ -91,6 +91,27 @@ protected override void Up(MigrationBuilder builder)
...
+      builder.CreateTable(
+        name: DbModuleSetting.TableName,
+        columns: table => new
+        {
+          Id = table.Column<Guid>(nullable: false),
+          Host = table.Column<string>(nullable: false),
+          Post = table.Column<int>(nullable: false),
```
Comment (MariyaSin) [inline src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs:101]: Port

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs [hunk 3]
```diff
@@ -91,6 +91,27 @@ protected override void Up(MigrationBuilder builder)
...
+          Email = table.Column<string>(nullable: false),
+          Password = table.Column<string>(nullable: false),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey($"PK_DbModuleSetting.TableName", x => x.Id);
```
Comment (MariyaSin) [inline src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs:112]: are you sure what your PK has the correct name?

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs [hunk 4]
```diff
@@ -91,6 +91,27 @@ protected override void Up(MigrationBuilder builder)
...
+        }
+        );
```
Comment (MariyaSin) [inline src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs:114]: move it in previous line

**Review by Cheephot — APPROVED** [review]

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+using System.Threading.Tasks;
+using LT.DigitalOffice.EmailService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.EmailService.Data.Provider.MsSql.Ef.Migrations
+{
+  [DbContext(typeof(EmailServiceDbContext))]
+  [Migration("20211611160500_CreateModuleSettings")]
+  class _20211611160500_CreateModuleSettings : Migration
```
Comment (actimellka) [inline src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs:14]: maybe public?

**Review by actimellka — APPROVED** [review]

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs (hunk 5 above)
Comment (irlirla) [reply src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs]: fixed

**Review by grinakate — APPROVED** [review]

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        constraints: table =>
+        {
+          table.PrimaryKey($"PK_{DbModuleSetting.TableName}", x => x.Id);
+        });
+    }
+
+    protected override void Down(MigrationBuilder builder)
+    {
+      builder.DropTable(
+          name: DbModuleSetting.TableName);
```
Comment (BelehovEgor) [inline src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs:42]: fix tab

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+using System.Threading.Tasks;
+using LT.DigitalOffice.EmailService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.EmailService.Data.Provider.MsSql.Ef.Migrations
+{
+  [DbContext(typeof(EmailServiceDbContext))]
+  [Migration("20211611160500_CreateModuleSettings")]
+  public class _20211611160500_CreateModuleSettings : Migration
```
Comment (BelehovEgor) [inline src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs:14]: remove data from name

Comment (BelehovEgor) [reply src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs:14]: name of class, name of file don't

**Review by BelehovEgor — APPROVED** [review]

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs (hunk 7 above)
Comment (irlirla) [reply src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211611160500_CreateModuleSettings.cs]: fixed

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs (hunk 1 above)
Comment (irlirla) [reply src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs]: fixed

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs (hunk 2 above)
Comment (irlirla) [reply src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs]: fixed

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs (hunk 3 above)
Comment (irlirla) [reply src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs]: fixed

File: src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs (hunk 4 above)
Comment (irlirla) [reply src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs]: fixed

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**